tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -96,6 +96,11 @@ public Project ConfigureProject()
                 {
                     AttributesDefinition = "Secure=yes"
                 },
+                // Set to "true" to allow the currently logged in user to be used as the Agent user.
+                new Property("ALLOW_CURRENT_USER")
+                {
+                    AttributesDefinition = "Secure=yes"
+                },
```

Comment:
I kind of agree with Julien....Here's a question: 

Do we think the number of customers that absolutely need to use their own account is more than the number of customers that may accidentally turn this on without understanding what it is?

Reply:
discussed offline.

We decided to remove the option because the steps required to make the option necessary to deploy the agent break too many Datadog and Microsoft recommendations.